pronotepy/ent/generic_func.py

```diff
@@ -412,8 +412,17 @@ def _oze_ent(
             if app["code"] == "pronote":
                 proxysso_url = urljoin(url, app["externalRoute"])
 
+        # If we still haven't got the url, try something else
```

Comment:
Where is the if statement checking if we already have the URL or not?

Reply:
I'm gonna do a quick new commit to fix this, while running black formatter